moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -568,14 +568,14 @@ void TrajectoryExecutionManager::updateControllerState(const std::string& contro
 
 void TrajectoryExecutionManager::updateControllerState(ControllerInformation& ci, const rclcpp::Duration& age)
 {
-  if (rclcpp::Clock().now() - ci.last_update_ >= age)
+  if (rclcpp::Clock(RCL_ROS_TIME).now() - ci.last_update_ >= age)
```

Comment:
Maybe a better readable way to get the current time is `Node::now()`. By default nodes are initialized with ros time.